### PR TITLE
By able to eayier build the project throw a proxy

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,5 @@
-FROM debian:stretch
-LABEL maintainer Camptocamp "info@camptocamp.com"
-
-RUN \
-  apt-get update && \
-  apt-get install --assume-yes --no-install-recommends gettext-base python3 && \
-  apt-get clean && \
-  rm --recursive --force /var/lib/apt/lists/*
+ARG VERSION
```
Comment (fredj) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile]: is it working to not have the `FROM` on the first line ?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile]: Yes :-)

Comment (arnaud-morvan) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile]: Yes, but only with a recent version of docker

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile]: It's released tow year ago... (from  17.05)

**Pull request merged**